partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnPlus.kt

```diff
@@ -89,8 +89,8 @@ internal val Fn_PLUS__DECIMAL_ARBITRARY_DECIMAL_ARBITRARY__DECIMAL_ARBITRARY = F
     name = "plus",
     returns = PType.decimal(),
     parameters = arrayOf(
-        @Suppress("DEPRECATION") Parameter("lhs", PType.decimal()),
-        @Suppress("DEPRECATION") Parameter("rhs", PType.decimal()),
+        Parameter("lhs", PType.decimal()),
```

Comment:
`FN` name var for above uses DECIMAL_ARBITRARY, perhaps needs to get changed?

Reply:
Eventually, yes. Currently rewriting all of the functions. See #1647